app/helpers/application_helper.rb

```diff
@@ -1,5 +1,9 @@
 module ApplicationHelper
-  def stars_list
-    [5,4,3,2,1].map { |number| [pluralize(number, "Star")] }
+  def options_for_video_reviews(selected=nil)
```

Comment:
I would recommend to use spaces around operators `=`. `selected = nil`, it is slightly more readable.